src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(/;\s*/g) // Розділяємо рядок на окремі CSS-правила за допомогою ";"
+    .filter(Boolean) // Видаляємо порожні елементи
+    .map((rule) => rule.split(/:\s*/))
```

Comment:
The regular expression `/\s*:\s*/` is used to split the rule into key-value pairs. Ensure that this regex correctly handles all possible whitespace scenarios around the colon.